ci_framework/roles/ci_setup/tasks/packages.yml

```diff
@@ -48,3 +48,20 @@
         remote_src: true
         mode: "0755"
         creates: "{{ cifmw_ci_setup_openshift_client_install_path }}/oc"
+
+- name: Inject oc completion in local profile
+  block:
+    - name: Create completion file
+      ansible.builtin.shell:  # noqa: risky-shell-pipe
+        cmd: >-
+          {{ cifmw_ci_setup_openshift_client_install_path }} completion bash |
+          tee -a ~/.oc_completion
+        creates: "{{ ansible_user_dir }}/.oc_completion"
```

Comment:
Nice!
>  creates path | A filename, when it already exists, this step will not be run.
[src](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#parameter-creates)

There is still a risk of running outdated `oc completion` with newer `oc`.
What do you think?

Reply:
this completion is done on the controller-0 - so it's a temporary host. If ppl update their openshift, they will redeploy it. And if really, they can regenerate the completion.